### PR TITLE
Pro 4734 check outdated deps

### PR DESCRIPTION
**Pull request by ETLaurent**
image

### CODE REVIEW

File: .github/workflows/outdated-dependencies.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 name: Check outdated dependencies
 on:
+  push:
+    branches: ["pro-4734-check-outdated-deps"]
```
Comment (ETLaurent) [inline .github/workflows/outdated-dependencies.yml:4]: rm

File: .github/workflows/outdated-dependencies.yml [hunk 2]
```diff
@@ -1,37 +1,43 @@
...
-          node-version: ${{ matrix.node-version }}
-      - id: run_outdated
-        run: npm outdated >> "$GITHUB_OUTPUT"
+          node-version: 18
+      - name: Install dependencies
+        run: npm install
+      - name: Check outdated dependencies
+        run: |
+          echo "$(npm outdated)" > output
+          npm outdated
```
Comment (ETLaurent) [inline .github/workflows/outdated-dependencies.yml:24]: way of getting the correct exit status

File: .github/workflows/outdated-dependencies.yml [hunk 3]
```diff
@@ -1,37 +1,43 @@
 name: Check outdated dependencies
 on:
-  # TODO: uncomment schedule
-  # schedule:
-  #   # Runs Mon-Fri at 0:00 UTC ( 00:00 EST )
-  #   - cron: "0 0 * * 1-5"
+  schedule:
+    # Runs Mon-Fri at 0:00 UTC ( 00:00 EST )
+    - cron: "0 0 * * 1-5"
```
Comment (boutell) [inline .github/workflows/outdated-dependencies.yml:5]: Once every two weeks would be more appropriate. We will never be 100% up to date with the same major version of everything, often up to date modules require older but maintained versions of other modules, etc. If we generate this report daily we will become blind to it (:

File: .github/workflows/outdated-dependencies.yml (hunk 2 above)
Comment (boutell) [reply .github/workflows/outdated-dependencies.yml:24]: `npm outdated > output` should do this by itself 🤔 Unless your goal was just to always add a newline. Not wrong though

**Review by boutell — CHANGES_REQUESTED** [review]

File: .github/workflows/outdated-dependencies.yml (hunk 2 above)
Comment (ETLaurent) [reply .github/workflows/outdated-dependencies.yml:24]: I wasn't able to keep the exit status when writing the output into the `output` file, so I'm running the same command again... not pretty but it works

File: .github/workflows/outdated-dependencies.yml (hunk 3 above)
Comment (ETLaurent) [reply .github/workflows/outdated-dependencies.yml]: Would every week do?
I found a syntax to run it every 2 week: `0 8 * * MON test $((10#$(date +\%W)\%2)) -eq 1`, but it's not compatible with GA

**Review by boutell — APPROVED** [review]
Once a week will have to do. 👍

**Pull request merged**